src/astro/sql/operators/cleanup.py

```diff
@@ -97,7 +97,15 @@ def _is_dag_running(self, task_instances: List[TaskInstance]) -> bool:
             (ti.task_id, ti.state)
             for ti in task_instances
             if ti.task_id != self.task_id
-            and ti.state not in [State.SUCCESS, State.FAILED, State.SKIPPED]
+            and ti.state
+            not in [
```

Comment:
@kaxil @dimberman - am I missing any terminal states here?

Reply:
@utkarsharma2 as far as I can tell this is all of them. We can make another PR if we find any more edge cases.